yt/utilities/command_line.py

```diff
@@ -152,6 +150,25 @@ def _get_girder_client():
     return gc
 
 
+class FileStreamer:
+    final_size = None
+    next_sent = 0
+    chunksize = 16*1024*1024
```

Comment:
16 MB is a bit too big of a chunksize I think. Right now the progress bar updates very slowly and won't even update once for smallish files. Testing locally, 100 kB seems to work pretty well and doesn't slow down the upload at all as far as I can see.